config/configmap/inferenceservice.yaml

```diff
@@ -40,9 +40,9 @@ data:
             "multiModelServer": "false"
         },
         "pytorch": {
-            "image": "gcr.io/kfserving/pytorchserver",
-            "defaultImageVersion": "v0.4.1",
-            "defaultGpuImageVersion": "v0.4.1-gpu",
+            "image": "pytorch/torchserve",
```

Comment:
Let's still keep the original pytorchserver for the time being and we can phase out in later version once torchserve integration is more mature. Here we can do the same strategy as [sklearnserver](https://github.com/kubeflow/kfserving/blob/master/config/configmap/inferenceservice.yaml#L26) is doing
```
        "pytorch": {
            "v1" : {
                "image": "gcr.io/kfserving/pytorchserver",
                "defaultImageVersion": "v0.5.0-rc0",
                "defaultGpuImageVersion": "v0.5.0-rc0-gpu",
                "supportedFrameworks": [
                    "pytorch"
                ],
                "multiModelServer": "false"
            },
            "torchserve" : {
                "image": "pytorch/torchserve",
                "defaultImageVersion": "0.2.0",
                "defaultGpuImageVersion": "0.2.0-gpu",
                "supportedFrameworks": [
                    "pytorch"
                ],
                "multiModelServer": "true"
            },
        },
```